notebooks/02_model/mmlspark_lightgbm_criteo.ipynb

```diff
@@ -80,7 +80,7 @@
     "    # get the maven coordinates for MML Spark from databricks_install script\n",
     "    from scripts.databricks_install import MMLSPARK_INFO\n",
     "    packages = [MMLSPARK_INFO[\"maven\"][\"coordinates\"]]\n",
-    "    repo = MMLSPARK_INFO[\"maven\"].get(\"repositories\", None)\n",
+    "    repo = MMLSPARK_INFO[\"maven\"][\"repo\"]\n",
```

Comment:
good catch, however, we have to add `.get("repo", None)`, if not spark function will break if we use something similar to what we have originally, where there was only coordinates.

Reply:
i wanted to force an error in that case, should we carry forward if they get out of synch?
also, fwiw if you use dict.get(), None is automatically provided as the default. [reference](https://docs.python.org/3/library/stdtypes.html#dict.get)

Reply:
> if you use dict.get(), None is automatically provided as the default. 

yeah, so the idea is if we are using the old dictionary, that doesn't have repo, the spark loader doesn't fail:

```
MMLSPARK_INFO = {"maven": {"coordinates": "Azure:mmlspark:0.16"}}
```
I guess that the dictionary that we have now is temporal:
```
MMLSPARK_INFO = {"maven": {"coordinates": "com.microsoft.ml.spark:mmlspark_2.11:0.16.dev8+2.g6a5318b",
                          "repositories": "https://mmlspark.azureedge.net/maven"}
                }
```

> i wanted to force an error in that case

don't understand very well what you are thinking, can you elaborate?

Reply:
I see so, you're just saying we shouldn't bother updating this notebook if we drop the repo piece in the future (which we're planning on doing), i'm fine with that.